content/en/getting_started/agent.md

```diff
@@ -0,0 +1,89 @@
+---
+title: Getting Started with the Agent
+kind: documentation
+further_reading:
+- link: "/agent/basic_agent_usage/"
+  tag: "Documentation"
+  text: "Basic Agent Usage"
+- link: "/agent/faq/why-should-i-install-the-agent-on-my-cloud-instances/"
+  tag: "FAQ"
+  text: "Why should I install the Datadog Agent on my cloud instances?"
+---
+
+## Overview
+The Agent is lightweight software installed on your hosts. It reports metrics and events from your host to Datadog. With additional setup, the Agent can report [live processes][1], [logs][2], and [traces][3].
```

Comment:
We should explain that metrics collection can happen through integrations and DogStatsD and link to the appropriate pages.